### PR TITLE
Update install-kubeadm.md

### PR DESCRIPTION
**Pull request by mxxisv**
This command is not in ubuntu, sync from English page

### CODE REVIEW

**Comment by mxxisv** [comment]
I signed it, please check.

File: content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md [hunk 1]
```diff
@@ -320,7 +320,7 @@ For more information on version skews, see:
 ```bash
 apt-get update && apt-get install -y apt-transport-https curl
 curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
-cat <<EOF >/etc/apt/sources.list.d/kubernetes.list
```
Comment (tanjunchen) [inline content/zh/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md]: see https://github.com/kubernetes/website/blob/master/content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md#installing-kubeadm-kubelet-and-kubectl

**Comment by mxxisv** [comment]
I signed it

**Comment by mxxisv** [comment]
I signed it

**Pull request merged**